src/Scene.cpp

```diff
@@ -15,7 +16,8 @@ Scene::Scene(sf::RenderWindow& window):
 
     updateGroup.push_back(player.get());
     updateGroup.push_back(frog.get());
-
+    
```

Comment:
and again my OCD kicks in - one newline would be enough :D sorry for nit-picks, you should make some bugs and mistakes next time so that I focus on them instead of having to be so pedantic about code conventions :D